bug: Project manual local setup is failing.

**Describe the bug**
Database seeding fails while running the setupDB.sh script while setting the project up locally.

**To Reproduce**
Steps to reproduce the behavior:
1. Follow the steps in the README.md to set the project locally (the one that doesn't use docker-compose)
2. After running the script ```setupDB.sh```, I got the error that the seeding has failed due to ``` Unique constraint failed on the fields: (`id`) ```

**Expected behavior**
The expected behavior was that the script would run and the docker container running Postgres that starts would have some dummy data. 

**Screenshots or GIFs**
![image](https://github.com/code100x/daily-code/assets/115103314/82403527-755c-466e-8941-96b703ba8a0a)


```bash
PrismaClientKnownRequestError: 
Invalid `db.track.create()` invocation in
/home/osatapathy/kirat-Cohort/cohort-1-to-100/week_20_openAPI/daily-code/packages/db/prisma/seed.ts:9:32

  6 const promises: Promise<any>[] = [];
  7 seedsData.forEach((seed) => {
  8   if (!hashID.includes(seed.data.id)) {
→ 9     const promise = db.track.create(
Unique constraint failed on the fields: (`id`)
    at In.handleRequestError (/home/osatapathy/kirat-Cohort/cohort-1-to-100/week_20_openAPI/daily-code/node_modules/@prisma/client/runtime/library.js:122:6854)
    at In.handleAndLogRequestError (/home/osatapathy/kirat-Cohort/cohort-1-to-100/week_20_openAPI/daily-code/node_modules/@prisma/client/runtime/library.js:122:6188)
    at In.request (/home/osatapathy/kirat-Cohort/cohort-1-to-100/week_20_openAPI/daily-code/node_modules/@prisma/client/runtime/library.js:122:5896)
    at async l (/home/osatapathy/kirat-Cohort/cohort-1-to-100/week_20_openAPI/daily-code/node_modules/@prisma/client/runtime/library.js:127:11167)
    at async Promise.all (index 5)
    at async main (/home/osatapathy/kirat-Cohort/cohort-1-to-100/week_20_openAPI/daily-code/packages/db/prisma/seed.ts:15:3) {
  code: 'P2002',
  clientVersion: '5.12.0',
  meta: { modelName: 'Track', target: [ 'id' ] }
}

An error occurred while running the seed command:
Error: Command failed with exit code 1: ts-node prisma/seed.ts
```

**Info (please complete the following information):**
 - Node.js : v22.3.0
 - Docker : v26.1.4
 - Docker-compose : v2.27.2

**Additional context**
My hypothesis is that the seed data has some duplicate id value and the quickest solution if that's the problem is, is to reupload the dummy data from the actual database.

hello @decoded-satapathy @Chaitanya1672, I was trying to setup the project locally for the first time, and got the following issue (attached below), my machine has postgres locally running on port 5432 so i've made relevant changes to connect the db to a docker postgres container.
`Asus@DESKTOP-5TBFKPS MINGW64 /d/my-git-contributions-folder/daily-code/packages/db (main)
$ ./setupDB.sh
=================DB Setup=================
.env file exists
Will you use a Local DB (L) or Cloud DB (C)
L
Will you use Postgres Server (P) or Docker Setup (D)
D
Setting up Docker
time="2024-06-29T11:29:41+05:30" level=warning msg="D:\\my-git-contributions-folder\\daily-code\\packages\\db\\docker-compose.yml: `version` is obsolete"
[+] Running 1/0
 ✔ Container db-postgres-1  Running                                                                  0.0s 
=================Container is up=================
=================Migrating DB=================
node:internal/modules/cjs/loader:1147
  throw err;
  ^

Error: Cannot find module 'D:\my-git-contributions-folder\daily-code\node_modules\node_modules\prisma\build\index.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
    at Module._load (node:internal/modules/cjs/loader:985:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v20.11.0
Seeds not applied. Proceeding to seed.
=================Seeding DB=================
node:internal/modules/cjs/loader:1147
  throw err;
  ^

Error: Cannot find module 'D:\my-git-contributions-folder\daily-code\node_modules\node_modules\prisma\build\index.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
    at Module._load (node:internal/modules/cjs/loader:985:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v20.11.0`

![Screenshot (35)](https://github.com/code100x/daily-code/assets/68049303/17ef1bc9-3139-4c43-a5d6-092ba7a62d85)

I am facing the same errors while using a cloud data base as well,
`Asus@DESKTOP-5TBFKPS MINGW64 /d/my-git-contributions-folder/daily-code/packages/db (main)
$ ./setupDB.sh
=================DB Setup=================
.env file exists
Will you use a Local DB (L) or Cloud DB (C)
C
Enter the DB URI of your Cloud DB
postgresql://test_owner:da8RqY4rDIX@ep-steep-waterfall-a5rgxxnl.us-east-2.aws.neon.tech/100xdevs?sslmode=require
=================Migrating DB=================
node:internal/modules/cjs/loader:1147
  throw err;
  ^

Error: Cannot find module 'D:\my-git-contributions-folder\daily-code\node_modules\node_modules\prisma\build\index.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
    at Module._load (node:internal/modules/cjs/loader:985:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v20.11.0
Seeds not applied. Proceeding to seed.
=================Seeding DB=================
node:internal/modules/cjs/loader:1147
  throw err;
  ^


Error: Cannot find module 'D:\my-git-contributions-folder\daily-code\node_modules\node_modules\prisma\build\index.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
    at Module._load (node:internal/modules/cjs/loader:985:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v20.11.0`

> hello @decoded-satapathy @Chaitanya1672, I was trying to setup the project locally for the first time, and got the following issue (attached below), my machine has postgres locally running on port 5432 so i've made relevant changes to connect the db to a docker postgres container. `Asus@DESKTOP-5TBFKPS MINGW64 /d/my-git-contributions-folder/daily-code/packages/db (main) $ ./setupDB.sh =================DB Setup================= .env file exists Will you use a Local DB (L) or Cloud DB (C) L Will you use Postgres Server (P) or Docker Setup (D) D Setting up Docker time="2024-06-29T11:29:41+05:30" level=warning msg="D:\\my-git-contributions-folder\\daily-code\\packages\\db\\docker-compose.yml: `version` is obsolete" [+] Running 1/0 ✔ Container db-postgres-1 Running 0.0s =================Container is up================= =================Migrating DB================= node:internal/modules/cjs/loader:1147 throw err; ^
> 
> Error: Cannot find module 'D:\my-git-contributions-folder\daily-code\node_modules\node_modules\prisma\build\index.js' at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15) at Module._load (node:internal/modules/cjs/loader:985:27) at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12) at node:internal/main/run_main_module:28:49 { code: 'MODULE_NOT_FOUND', requireStack: [] }
> 
> Node.js v20.11.0 Seeds not applied. Proceeding to seed. =================Seeding DB================= node:internal/modules/cjs/loader:1147 throw err; ^
> 
> Error: Cannot find module 'D:\my-git-contributions-folder\daily-code\node_modules\node_modules\prisma\build\index.js' at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15) at Module._load (node:internal/modules/cjs/loader:985:27) at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12) at node:internal/main/run_main_module:28:49 { code: 'MODULE_NOT_FOUND', requireStack: [] }
> 
> Node.js v20.11.0`
> 
> ![Screenshot (35)](https://private-user-images.githubusercontent.com/68049303/344371500-17ef1bc9-3139-4c43-a5d6-092ba7a62d85.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk4MTIwMTIsIm5iZiI6MTcxOTgxMTcxMiwicGF0aCI6Ii82ODA0OTMwMy8zNDQzNzE1MDAtMTdlZjFiYzktMzEzOS00YzQzLWE1ZDYtMDkyYmE3YTYyZDg1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzAxVDA1MjgzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE3NDJjYmQzNDc4MTQ5NjcwZWUxNTkxZjI4NWE4ZWI5YTFjM2Y4MjQ2NzUyODY1OWQ5ZGY1YjQ5N2FiYjA3OWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.HVW1WmFyEAKdffHCbUmslpxA4qAD4HU-aN16ySgjPXM)

i was able to fix the issue. This issue is faced by windows users and is discussed in issue #390, dumb of me not to look further 🤦

--- a/README.md
+++ b/README.md
@@ -15,12 +15,20 @@ cd daily-code
 yarn install
 ```
 
-> Setup DB (Windows users must have git-bash installed to run the script)
+> Setup DB 
 
+For Mac and Linux users
 ```
 cd packages/db
 chmod +x ./setupDB.sh
 ./setupDB.sh
+```
+
+For windows users
+```
+cd packages/db
+yarn prisma migrate dev
+yarn prisma db seed
 ```
 
 > Run locally

--- a/README.md
+++ b/README.md
@@ -24,7 +24,7 @@ chmod +x ./setupDB.sh
 ./setupDB.sh
 ```
 
-For windows users
+For Windows users
 ```
 cd packages/db
 yarn prisma migrate dev

